tests/performance/test_redis_mongo_benchmark.py

```diff
@@ -129,10 +128,10 @@ def print_results(self):
         for k, v in self.test_duration.items():
             if len(v) > 1:
                 PERFORMANCE_REPORT.append(
-                    f'{k}\tAverage: {statistics.mean(v)}\tSTDEV: {statistics.stdev(v)}'
+                    f'{k}\tAverage: {statistics.mean(v) / 1000000000}\tSTDEV: {statistics.stdev(v) / 1000000000}'
```

Comment:
Would rather have a constant, than a magic number, even for simple conversions like this.
Shouldn't we use the _ns() instead that already gives us time in nanoseconds?